airquality/README.md

```diff
@@ -0,0 +1,65 @@
+Provided by: Kristina Robinson
+
+Air Quality (Ozone and Small Particulates per Million) by zipcode from the U.S. EPA AirNow.gov API.
+
+Quick Install
+
+InfluxDB UI
+
+In the InfluxDB UI, go to Settings-> Templates and enter this URL: http://raw.githubusercontent.com/influxdata.community-templates.master/airquality/airquality.yaml
+
+InfluxCLI
+
+If you have your InfluxDB credentials configured in teh CLI, you can install this template with: 
+```influx apply -u https://rawgithubusercontent.com/influxdata/community-templates/master/airquality/airquality.yaml```
+
+
+Included Resources
+
+* 1 Bucket: `airquality`, 30 day retention (or you may use your own bucket)
+* 1 Label: `airquality`
+* 1 Telegraf Configuration `airquality-telegraf-config`
+* 1 Dashboard: `Air Quality`
+* 1 Variable: `bucket` (allows any bucket to be used)
+
+Setup Instructions
+
+Go to https://docs.airnowapi.org/account/request/ to request account access.
+Go to https://docs.airnowapi.org/forecastsbyzip/query, and build your URL as follows:
+* Enter the zip code of interest
+* Select `application/json` from the Format dropdown
+* Click `Build`
+* Copy the URL string.
+Inside InfluxDB, click Data, Telegraf and edit the airquality-telegraf-config file.  Under the `[[inputs.http]]` section, paste the url value copied from the previous step where it says "CHANGEME".
```

Comment:
It would be easier to make this an envvar and add an instruction to set it with the `INFLUX_` envvars below

Reply:
OK, I will add this as an environment var as well.